### PR TITLE
feat: mimetype for REST controller response

### LINKED ISSUE #1272: REST controllers response mimetype
mime-type in the response from REST controllers should be application/json.

### PR DESCRIPTION
**Pull request by sudeshrshetty**
- added `application/json` mimetype for REST controller response
- closes #1272

Signed-off-by: sudesh.shetty <sudesh.shetty@securekey.com>

### CODE REVIEW

File: pkg/controller/rest/rest.go [hunk 1]
```diff
@@ -59,7 +61,6 @@ func SendError(rw http.ResponseWriter, err command.Error) {
 // SendHTTPStatusError sends given http status code to response with error body
 func SendHTTPStatusError(rw http.ResponseWriter, httpStatus int, code command.Code, err error) {
 	rw.WriteHeader(httpStatus)
-	rw.Header().Set("Content-Type", "application/json")
```
Comment (troyronda) [inline pkg/controller/rest/rest.go]: why is this removed?

Comment (sudeshrshetty) [reply pkg/controller/rest/rest.go]: line #35 always gets called, so it gets set twice in case of errors
